go.mod

```diff
@@ -106,5 +105,4 @@ require (
 replace (
 	github.com/coreos/go-oidc => github.com/gravitational/go-oidc v0.0.3
 	github.com/iovisor/gobpf => github.com/gravitational/gobpf v0.0.1
-	github.com/sirupsen/logrus => github.com/gravitational/logrus v0.10.1-0.20171120195323-8ab1e1b91d5f
```

Comment:
Do you know why we made a fork?
Hope we're not losing some internal patch that teleport depends on.

Reply:
Awesome, hopefully we can archive our fork after this merges!

Reply:
@a-palchikov With this change will `WithError(err)` finally start behaving properly by printing the error with a stack trace, or would we need to additionally bump logrus to get it working?